examples/index.json

```diff
@@ -536,7 +536,7 @@
                 "isLearnByExample": true
             },
             {
-                "name": "Ordering",
+                "name": "Order",
```

Comment:
I don't think this makes sense. "Ordering" here is used as a noun.

Reply:
Reverted to `Ordering`.